README.md

```diff
@@ -59,7 +59,31 @@ There are three ways to install ape: `pipx`, `pip`, or `Docker`.
 
 Ape can also run in a docker container.
 
-Please visit our [Dockerhub](https://hub.docker.com/repository/docker/apeworx/ape) for more details on using Ape with Docker.
+You can pull our images from [ghcr](https://ghcr.io/apeworx/ape). We currently have a `slim` docker image as well as an image that is built with the recommended-plugins.
+
+You can pull the image:
+
+```bash
+docker pull ghcr.io/apeworx/ape:latest-slim  # installs ape with required packages
+```
+
+or pull with the recommended-plugins:
```

Comment:
I think put the other image first, and not recommended to install w/ `slim` unless you need to

Reply:
lmk if that's what you're looking for